recipes/ttyd/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ttyd" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tsl0922/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b5b62ec2ce08add0173e6d1dfdd879e55f02f9490043e89f389981a62e87d376
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script:
+    - mkdir build
+    - cd build
+    - cmake -DCMAKE_INSTALL_PREFIX={{PREFIX}} -DCMAKE_PREFIX_PATH={{PREFIX}} ..
+    - make 
+    - make install
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake 
+    - pkg-config
+    - vim
```

Comment:
Should vim be a host dep?

Reply:
Not in this case. The project has a dependency on the `xxd` executable during build time only, not a link dependency.

Reply:
The `ttyd` [CMakeLists.txt][link] file uses `xxd` only during build time. This is lifted from their code.

```cmake
find_program(CMAKE_XXD NAMES xxd)
add_custom_command(OUTPUT html.h
        COMMAND ${CMAKE_XXD} -i index.html html.h
        WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/src
        COMMENT "Generating html.h from index.html")
list(APPEND SOURCE_FILES html.h)
```
[link]: https://github.com/tsl0922/ttyd/blob/01a2ce07257bc5f6be02e8b870f6eab0d03c8d23/CMakeLists.txt#L62-L67

Reply:
Doesn't that make it a host dep?

Reply:
I don't think so, according to [this post][post] by isuruf, you should move something to host if it is used as a library, which is not the case here.

The way I understood it from the [documentation][doc], we put a tool in `build` if we need it executed during build time, but it won't be needed by the end-package during runtime, e.g. cmake.

[doc]: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#build
[post]: https://github.com/conda-forge/staged-recipes/pull/9136#discussion_r315531004

Reply:
Ahh I see it doesn't contain a header file but is only used to generate the header. Yeah this is fine.